wordpress-mobile/WordPress-iOS
# [Gutenberg] Fix pressable image sizes

Fixes issue described on https://github.com/WordPress/gutenberg/pull/18627#issuecomment-556184588

To test:
- Add a Pressable site to the self-hosted sites. (Do not sign up to Jetpack)
- Start a new post on that site.
- Add an image block.
- Add an image to the image block.
- Check that the Image URL is properly set for the Image Size slug (Large by default)
  - The image URL needs to have the image size on its url: `...-768x1024.jpg`
- Try other size options and check that the image url changes.

PR submission checklist:
